plugins.rst

```diff
@@ -19,23 +19,29 @@ Compared to adding regular software, plugins offer the following benefits:
 Overview
 --------
 
-The following is the list of official plugins by |omv|.
+The following is the list of official plugins by |omv|. Their availability depends on the architecture on which |omv| is installed and/or, for some, if Podman is installed.
+Podman container-based plugins require Podman to be installed, either manually or through the Podman plug-in.
```

Comment:
This is wrong. Podman is a package dependency of the plugins that rely on it.
Please remove that sentence.

Reply:
Than I've got a different issue because of which the Podman container plugins, except the Podman one itself, are not showing up for me. I'll have a look at that later on.